packages/generator/src/generator.spec.ts

```diff
@@ -42,11 +45,11 @@ describe('generator', () => {
         outputDir: 'common',
         overwrite: true,
         prettierConfig: '/prettier/config',
-        generateSdkMetadata: true,
+        metadata: true,
         include: join(pathTestResources, '*.md')
       });
       // TODO the first call will go away once ts-morph is removed
-      project = await generateProject(options);
+      project = await generateProject(createParsedOptions(options));
```

Comment:
[q] Can we not call `createParsedOptions` directly in ln 43?

Reply:
No because the `generate()`in line 53 needs the unparse options.

Reply:
Right, I missed that.